iotdb-core/datanode/src/main/java/org/apache/iotdb/db/queryengine/plan/scheduler/load/LoadTsFileScheduler.java

```diff
@@ -141,6 +143,7 @@ public LoadTsFileScheduler(
     this.allReplicaSets = new HashSet<>();
     this.isGeneratedByPipe = isGeneratedByPipe;
     this.block = LoadTsFileMemoryManager.getInstance().allocateDataCacheMemoryBlock();
+    this.loadTsFileRateLimiter = new LoadTsFileRateLimiter();
```

Comment:
Make the limiter private per scheduler may not be of use because the scheduler is created per statement. It is maybe ideal to make a public static one in loadTsFileManager and use that.